docs/package_templates/autotools_package/all/conanfile.py

```diff
@@ -143,8 +141,10 @@ def generate(self):
         tc = PkgConfigDeps(self)
         tc.generate()
         # generate dependencies for autotools
-        tc = AutotoolsDeps(self)
-        tc.generate()
+        # some recipes might require a workaround for MSVC:
+        # https://github.com/conan-io/conan-center-index/blob/00ce907b910d0d772f1c73bb699971c141c423c1/recipes/xapian-core/all/conanfile.py#L106-L135
```

Comment:
This is an unfortunately rather frequently required workaround. I think it's beneficial to include it as (a) the ./configure error is just a cryptic "cannot create C executables" and (b) the fix is not at all obvious.

Reply:
Please, add a related issue too to be more visible about that idiom and will be done in the future.

Reply:
Done.